common/redis/templates/_helpers.tpl

```diff
@@ -12,31 +12,6 @@ We truncate at 63 chars because some Kubernetes name fields are limited to this
 {{- printf "%s-%s" .Release.Name $name | trunc 63 | replace "_" "-" | trimSuffix "-" -}}
 {{- end -}}
 
-{{/*
-  The (contains "." $repo) checks if the chart user has overridden the
-  repository to refer to a custom registry. If not, the default Docker Hub
-  mirror gets used.
-*/}}
-{{- define "redis.image" -}}
-  {{- $repo := required ".Values.image.repository missing" .Values.image.repository -}}
-  {{- $tag := required ".Values.image.tag missing" .Values.image.tag -}}
-  {{- if contains "." $repo -}}
```

Comment:
What? If the repo contains a dot, we are just not using the mirror!?...

Reply:
That's for image refs that already contain a hostname, e.g. Keppel forcing the alternate region mirrors.